plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -5485,7 +5484,8 @@ protected OptionalInt maxTableNameLength()
     {
         // This value depends on metastore type
         // The connector appends uuids to the end of all table names
-        return OptionalInt.of(255 - UUID.randomUUID().toString().length());
+        // 33 is the length of random suffix. e.g. {table name}-142763c594d54e4b9329a98f90528caf
+        return OptionalInt.of(255 - 33);
```

Comment:
why this change?

Reply:
Previously, this test passed in Iceberg unintentionally because of it's try-catch style. This change is required to adjust to the right max length.